test/utils/include/Partition_Test.cpp

```diff
@@ -0,0 +1,69 @@
+#include "gtest/gtest.h"
+#include <stdio.h>
+
+#include <boost/algorithm/string.hpp>
+#include <boost/lexical_cast.hpp>
+
+#include "utilities/partition/mpi/partition.hpp"
+
+class PartitionsParserTest: public ::testing::Test {
+
+    protected:
+
+    PartitionsParserTest() {
+
+    }
+
+    ~PartitionsParserTest() override {
+
+    }
+
+    void SetUp() override;
+
+    void TearDown() override;
+
+    void setupArbitraryExampleCase();
+
+};
+
+void PartitionsParserTest::SetUp() {
+    setupArbitraryExampleCase();
+}
+
+void PartitionsParserTest::TearDown() {
+
+}
+
+void PartitionsParserTest::setupArbitraryExampleCase() {
+
+}
+
+TEST_F(PartitionsParserTest, TestFileReader)
+{
+  const std::string file_path = "/home/shengting.cui/MPI/ngen/data/example_partition_config.json";
```

Comment:
Another absolute path.

Reply:
The previous test code might be an older version of this one. I'll check and may be delete that one.